backend/api/Helpers/Logging/LoggerExtension.cs

```diff
@@ -41,7 +41,7 @@ private static ElasticsearchSinkOptions ConfigureElasticSink(IConfigurationRoot
             {
                 ModifyConnectionSettings = x => x.BasicAuthentication(
                     configuration["Elastic:Username"],
-                    configuration["Elastic:Password"]),
+                    Environment.GetEnvironmentVariable("Elastic__Password")),
```

Comment:
Not sure why you need to change this?  This would indicate something is wrong with configuration.

Reply:
Somehow this was not getting the `Elastic__Password` from the Kubernetes/Openshift environment variables.